sound/soc/codecs/rt1011.c

```diff
@@ -1848,8 +1850,8 @@ static int rt1011_set_tdm_slot(struct snd_soc_dai *dai,
 	}
 
 	/* Rx slot configuration */
-	rx_slotnum = hweight_long(rx_mask);
-	first_bit = find_next_bit((unsigned long *)&rx_mask, 32, 0);
+	rx_slotnum = hweight_long(rx_mask_l);
+	first_bit = find_next_bit(&rx_mask_l, 32, 0);
```

Comment:
`find_next_bit()` is really for finding set bits in an array of longs. Here I'd just use `__ffs(rx_mask_l)`